### PR TITLE
Feature/admin post service test

### PR DESCRIPTION
**Pull request by murtazamehmudji**

### CODE REVIEW

File: src/common/constants/post.constants.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export enum POST_STATUS {
-  APPROVED = 'approved',
-  DECLINE = 'decline',
-  PENDING = 'pending',
+  APPROVED = 'APPROVED',
+  DECLINED = 'DECLINED',
+  PENDING = 'PENDING',
 }
```
Comment (miladezzat) [inline src/common/constants/post.constants.ts:5]: why, you change that?

File: src/marketplace/marketplace-post/entities/marketplace-post.entity.ts [hunk 2]
```diff
@@ -29,8 +30,8 @@ export class MarketplacePost {
   @Field()
   price: number;
 
-  @Field()
-  status: string;
+  @Field(type => POST_STATUS)
```
Comment (miladezzat) [inline src/marketplace/marketplace-post/entities/marketplace-post.entity.ts:33]: `type` not need

**Review by miladezzat — CHANGES_REQUESTED** [review]
@murtazamehmudji 
One Note: please when you made a PR focus on one thing

File: src/common/constants/post.constants.ts (hunk 1 above)
Comment (murtazamehmudji) [reply src/common/constants/post.constants.ts:5]: Because graphql reads the keys while creating enums. Thus, in enum keys their values need to be same.

**Review by miladezzat — APPROVED** [review]

**Pull request merged**